pandas/_libs/window/aggregations.pyx

```diff
@@ -1491,8 +1491,7 @@ cdef ndarray[float64_t] _roll_weighted_sum_mean(float64_t[:] values,
         tot_wgt = np.zeros(in_n, dtype=np.float64)
 
     if minp > win_n:
-        raise ValueError(f"min_periods (minp) must be <= "
-                         f"window (win)")
+        raise ValueError(f"min_periods {minp} must be <= window {win_n}")
```

Comment:
The ```f``` did nothing before. should I revert and just delete the ```f```? 
or leave it as it is?

Reply:
I _think_ leave the message unchanged unless obvious improvement and then should add a brief note in whatsnew that message has been changed.

Reply:
I think I'll just delete this ```f``` because it didn't do anything any.